### DISCUSSION TITLE
Next <Code><Br>, and Vote For My AI Demo!

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#35] [Comments]
# Next <Code><Br>, and Vote For My AI Demo!

An update on when you can expect the next <Code><Br>, and my submission to an AI Contest.

https://www.raymondcamden.com/2024/09/04/next-codebr-and-vote-for-my-ai-demo

**Comment by projectmb** [comment]
Content Assistant - I get the message Video is private. Well, I vote anyway
.

**Reply by cfjedimaster** [reply]
Hey I'm sorry - I realized I had made the video without audio. I corrected it here: https://www.youtube.com/watch?v=kOGcn_1DDT8

I've reached out to Google to ask them to update the link.
